proposals/lip-0065.md

```diff
@@ -109,9 +109,10 @@ To summarize, we chose sparse Merkle trees because:
 | `EVENT_TOPIC_BEFORE_TRANSACTIONS` | bytes | `0x02` | Default topic for events emitted _before_ transactions execution. |
 | `EVENT_TOPIC_AFTER_TRANSACTIONS` | bytes | `0x03` | Default topic for events emitted _after_ transactions execution. |
 | `EVENT_NAME_DEFAULT` | string | `commandExecutionResult` | Name of default event emitted at the end of transaction execution. |
-| `EVENT_INDEX_LENGTH_BITS` | integer | TBD | Length in bits of the index used to calculate events tree leaf keys. |
+| `EVENT_INDEX_LENGTH_BITS` | integer | `30` | Length in bits of the index used to calculate events tree leaf keys. |
 | `MAX_EVENTS_PER_BLOCK` | integer | `2^EVENT_INDEX_LENGTH_BITS` | Max number of events emitted per block. |
-| `TOPIC_HASH_LENGTH_BYTES` | integer | TBD | Output length in bytes of the hash function used to calculate events tree leaf keys. |
+| `MAX_EVENT_INDEX` | integer | `2^EVENT_INDEX_LENGTH_BITS - 1` | Max value of an event index. |
+| `TOPIC_HASH_LENGTH_BYTES` | integer | `8` | Output length in bytes of the hash function used to calculate events tree leaf keys. |
 | `TOPIC_INDEX_LENGTH_BITS` | integer | `2` | Length in bits of the topic index used to calculate events tree leaf keys. |
 | `MAX_TOPICS_PER_EVENT` | integer | `2^TOPIC_INDEX_LENGTH_BITS` | Max number of topics per event. |
 | `TOTAL_INDEX_LENGTH_BYTES` | integer | `(EVENT_INDEX_LENGTH_BITS+TOPIC_INDEX_LENGTH_BITS)/8` | Length in bytes of the concatenation of the event index and the topic index. |
```

Comment:
This definition implicitly assumes that the sum of `EVENT_INDEX_LENGTH_BITS` and `TOPIC_INDEX_LENGTH_BITS` is a multiple of 8. 

Not sure this is important, since I guess we hard code those values and the constants are not configurable by devs (am I right?). We could just add a comment that the sum of the two constants should always be a multiple of 8, just to be precise.